airflow/cli/commands/task_command.py

```diff
@@ -169,8 +169,6 @@ def task_run(args, dag=None):
 
     task = dag.get_task(task_id=args.task_id)
     ti = TaskInstance(task, args.execution_date)
-    ti.refresh_from_db()
```

Comment:
All methods (_run_task_by_executor, _run_task_by_local_task_job, _run_raw_task) will call refresh_from_db before execution, so it seems there's no reason to do this here.

Reply:
Does init_run_context look at any attributes of the TI? If so they won't be populated until this call.

Reply:
No, it only setups logging context :)

Reply:
This actually broke task logging for multiple tries, see 11717, because setting up logging _looks at the attributes of the TI_, via the `[logging].log_filename_template` setting.